src/global-settings.php

```diff
@@ -815,6 +818,25 @@ public function str_replace_first( $search, $replace, $subject ) {
 			}
 			return $subject;
 		}
+
+		/**
+		 * Prevent global settings from affecting the styles of the native Post
+		 * block. This fixes the issue where the last column of the native Posts
+		 * block incorrectly wraps below and leaves a gap.
+		 *
+		 * @param array $selectors
+		 * @param string $tag
+		 * @return void
+		 */
+		public function posts_block_columns_fix( $selectors, $tag ) {
+			// Prevent global settings from affecting hte native wp block post.
+			if ( $tag === 'li' ) {
+				$index = array_search( '[data-block-type="core"] li', $selectors );
```

Comment:
@prconcepcion Just realized that the next line will produce an error if the `array_search` fails, you need to make sure that the next line only runs when `array_search` succeeds